src/api/app/assets/stylesheets/webui/application/announcements.scss

```diff
@@ -0,0 +1,30 @@
+// Place all the styles related to the announcements controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+#announcements {
+  .announcement {
+    background-color: #fff6ef;
+    border: 1px solid #d2702e;
+    margin-bottom: 15px;
+
+    h3 {
+      margin: 0;
+      color: #d2702e;
+    }
+
+    p {
+      margin-bottom: 10px;
+      font-size: 0.9em;
+      color: #666;
+
+      a {
+        font-size: 0.8em;
+        color: #069;
+        &:hover { color: #690; }
```

Comment:
Is this really needed? The links should already be green on hover If not, this should be `$obs_green` from [colors.scss](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/stylesheets/webui2/colors.scss)

Reply:
But colors.scss has been included for webui2, this solution is only for old webui.
My plan is doing webui2 version when everything is approved.